internal/web/server.go

```diff
@@ -171,13 +171,14 @@ func RunServer(ctx context.Context, c cluster.Cluster,
 
 		// Successfully created all components with the new configuration.
 		confVersion = conf.Version
-		conf = nil // Clear conf to receive a new one in the next iteration.
 
 		// Create new handler.
 		newHandlerCtx, cancelNewHandlerCtx := context.WithCancel(context.Background())
 		newHandler, newHandlerStopped := NewHandler(newHandlerCtx, conf, spaHandler, kubeClient,
 			version, statusManager, namespace, reportInterval, eventRecorder, authMiddleware, serverLog)
 
+		conf = nil // Clear conf to receive a new one in the next iteration.
```

Comment:
Ouch! Sorry for that, hope it was not hard to debug

Reply:
I couldn't believe `system:masters` can not call the actions, then I realised what's going on, took some minutes :)))